pebbles/config.py

```diff
@@ -149,6 +149,8 @@ class BaseConfig(object):
 
     PROVISIONING_NUM_WORKERS = 1
 
+    CSC_LOGIN = 'change_me'
```

Comment:
Should we have AUTH_METHOD somewhere in the var name, maybe CSC_LOGIN_AUTH_METHOD or CSC_LOGIN_URL?